src/rules/no-unused-private-members/index.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+const { utils } = require("stylelint");
+const namespace = require("../../utils/namespace");
+const ruleUrl = require("../../utils/ruleUrl");
+
+const ruleName = namespace("no-unused-private-members");
+
+const messages = utils.ruleMessages(ruleName, {
+  expected: variable => `"${variable}" is defined but never used.`
```

Comment:
This message should be formatted:

```
"Expected [something] [in some context]"
```

https://stylelint.io/developer-guide/rules/#add-problem-messages

Reply:
Also, instead of `variable` the argument could be called `privateMember` (or something similar).